network/src/transport.rs

```diff
@@ -218,12 +218,10 @@ pub fn build_memory_noise_transport(
     own_handshake.add(SUPPORTED_MESSAGING_PROTOCOL, application_protocols);
 
     memory_transport
-        .and_then(move |socket, _addr, origin| async move {
+        .and_then(move |socket, addr, origin| async move {
             let (remote_static_key, socket) =
                 noise_config.upgrade_connection(socket, origin).await?;
-
-            // TODO(philiphayes): reenable after seed peers are always fully rendered
-            // expect_noise_pubkey(&addr, remote_static_key.as_slice(), origin)?;
+            expect_noise_pubkey(&addr, remote_static_key.as_slice(), origin)?;
```

Comment:
btw the name of this function makes me think of `expect` which panics. I'm not suggesting any changes though because the Noise IX -> IK PR will remove this check I believe.

Reply:
right, should rename to `verify_noise_pubkey_present` or something. `expect` == panic in my mind to, which is what it was originally to make tests fail faster : p

should be removed soon anyway